### ISSUE TITLE
Add configuration for warnings

### ISSUE THREAD

**Issue by AlexLandau** [AlexLandau/griddle#3]
Warnings (and possibly some current errors) should be able to be configurable to appear as errors or be ignored.

**Comment by arr28** [comment]
Yes please.  With this plugin and without the ability to configured these things, ggp-base has 16 errors, in various files that are used to test the GDL parsing / propnet generation code.
- test_case_3b.kif
- test_case_3e.kif
- test_case_3f.kif
- test_clean_not_distinct.kif

Which is a bit of a shame, because otherwise it builds without error.  However, if I delete the files in question, I can't run the tests.  And I dare say that if I fixed up the files, they'd no longer be testing what they're meant to be testing.

**Comment by AlexLandau** [comment]
Those mostly involve real issues like unsafe rules. You'd probably want to keep them as errors for anything else you're working on (and for many cases I wouldn't make them configurable; obvious errors are obvious errors).

I suppose there could be a mode to turn all Griddle problems into warnings, and just make the games you're working on warning-free.

My recommendation in the meantime would be to turn GDL validation off when not using it.

Another possibility would be to store files with intentionally invalid GDL using an extension other than .kif or .gdl, so Griddle doesn't see it.

**Comment by arr28** [comment]
Ah - I like the idea of storing deliberately bad GDL with a different
extension.  I'll do that.
